index.html

```diff
@@ -424,13 +424,9 @@ <h2>Creating LDP-NRs with HTTP POST</h2>
       <section id="http-put">
         <h2>HTTP PUT</h2>
         <p>
-          A <code>PUT</code> request against an existing resource MAY include a <code>rel="type"</code> link in the
-          <code>Link</code> header. If the type in the <code>Link</code> header is a subtype of a current type of the
-          resource, and has an interaction model assigned to it by [[!LDP]], then the resource MUST be assigned
-          the new type and the
-          interaction model of the resource MUST be changed to the interaction model assigned to the new type by
-          [[!LDP]]. If that type is a value in the LDP namespace and is not either a current type of the resource or a
-          subtype of a current type of the resource, the request MUST be rejected with a 409 (Conflict) response.
+          Impementations MAY allow the interaction model of an existing resource to be changed by specification of a
+          new LDP type in an HTTP <code>Link: rel="type"</code> header. If supported, requests SHOULD be rejected
+          with a 409 (Conflict) response unless the new LDP type specified is a subtype the resource's current type.
```

Comment:
Minor wording issue: can you add an "of" after subtype so it reads "is a subtype of the..."?

Reply:
ping @dannylamb ... well spotted @escowles